src/main/java/uk/ac/cam/cl/dtg/isaac/dto/users/UserSummaryWithEmailAddressAndGenderDTO.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2017 James Sharkey
+ * <br>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * <br>
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <br>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.cam.cl.dtg.isaac.dto.users;
+
+/**
+ * View of a User Summary object, which contains additional information (email).
+ * Usually used where a sharing relationship has or will be created.
+ */
+public class UserSummaryWithEmailAddressAndGenderDTO extends UserSummaryDTO {
```

Comment:
Is the naming of this DTO going to be confusing with "UserSummaryWithEmailAddressDTO"? 

Is the idea that we return the address and gender for every user with the API call and get the front-end to do the heavy lifting so it shows as ![image](https://github.com/isaaccomputerscience/isaac-api/assets/82041490/ff50c8bd-6439-43f3-b09b-7492804eb7ab) rather than just a list of genders as  string, as appears to be returned here?

Did you clear that return object with FE dev?

Reply:
I would have to disagree the DTO naming is very clear it admits to having User Summary information plus email address and gender. The reason behind creating a new DTO is because the DTO UserSummaryWithEmailAddressDTO is also used elsewhere and we didn't want to include an extra details when the call is used elsewhere